bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/AudioManager.java

```diff
@@ -72,23 +118,33 @@
      * @param sinkId The id of the audio sink to use or null
      * @throws AudioException in case the url stream cannot be opened
      */
-    void stream(String url, String sinkId) throws AudioException;
+    void stream(@Nullable String url, @Nullable String sinkId) throws AudioException;
+
+    /**
+     * Stream audio with the given volume from the passed url to the given sink
+     *
+     * @param url The url to stream from or null if streaming should be stopped
+     * @param sinkId The id of the audio sink to use or null
+     * @param volume The volume to be used or null if the default notification volume should be used
+     * @throws AudioException in case the url stream cannot be opened
+     */
+    void stream(@Nullable String url, @Nullable String sinkId, @Nullable PercentType volume) throws AudioException;
 
     /**
      * Retrieves the current volume of a sink
      *
-     * @param sinkId the sink to get the volume for
+     * @param sinkId the sink to get the volume for or null
      * @return the volume as a value between 0 and 100
      */
-    PercentType getVolume(String sinkId);
+    PercentType getVolume(@Nullable String sinkId);
```

Comment:
Why Nullable? What is returned if null is passed?

Reply:
You can use `null` to get the volume of the default audio sink. I will clarify it in the JavaDoc.